packages/shared/src/components/modals/SharedBookmarksModal.tsx

```diff
@@ -119,7 +119,7 @@ export default function SharedBookmarksModal({
             <DiscordIcon />
             <TwitterIcon />
             <SlackIcon />
-            <GithubIcon />
+            <GithubIcon size="medium" />
```

Comment:
Why is this one specifically medium?

Reply:
Other icons are .svg and size is set by the wrapping di, but this one is a component, so I need to explicitly set size for it

Reply:
Can we not convert them all to components then?

Reply:
Yes, I just made it